conf/messages

```diff
@@ -345,27 +257,24 @@ serialization.failed=Failed to serialize data.
 initialData.notEnabled=Initial Data Insertion is not enabled in the configuration of this wK instance
 initialData.organizationsNotEmpty=There are already organizations present in the database. Please refresh the db schema and try again
 
-job.notFound = Job with id {0} could not be found.
 job.couldNotRunCubing = Failed to start WKW conversion job.
 job.couldNotRunTiffExport = Failed to start Tiff export job.
 job.couldNotRunComputeMeshFile = Failed to start mesh file computation job.
 job.couldNotRunNucleiInferral = Failed to start nuclei inferral job.
 job.couldNotRunNeuronInferral = Failed to start neuron inferral job.
 job.couldNotRunGlobalizeFloodfills = Failed to start job for globalizing floodfills.
-job.couldNotRunApplyMergerMode = Failed to start job to apply merger mode tracing.
+job.couldNotRunApplyMergerMode = Failed to start job to apply merger mode annotation.
 job.disabled = Long-running jobs are not enabled for this webKnossos instance.
 jobs.worker.notFound = Could not find this worker in the database.
 job.export.fileNotFound = Exported file not found. The link may be expired.
 job.export.tiff.invalidBoundingBox = The selected bounding box could not be parsed, must be x,y,z,width,height,depth
 job.export.tiff.volumeExceeded = The volume of the selected bounding box is too large.
 job.export.tiff.edgeLengthExceeded = An edge length of the selected bounding box is too large.
-job.export.notAllowed.organization = Currently tiff export is only allowed for datasets of your own organization.
```

Comment:
I feel like this should not be unused 🤔 could you have a quick look in the jobs controller? Should be analogous to the other jobs (see lines directly below here)

Reply:
Looks like this was replaced with very specific responses instead. This general form is not used anywhere as far as I can tell with string search.

<img width="571" alt="image" src="https://user-images.githubusercontent.com/1105056/200509010-4c374e59-ddeb-4fd6-8b95-0b1542e2b437.png">

Reply:
Ok, apparently this was intentional after all, see https://github.com/scalableminds/webknossos/pull/6219 – that means the tiff-export-specific message was indeed intentionally unused :+1: